### PR TITLE
add remainder of delivery manager pages

### PR DESCRIPTION
**Pull request by backslashbaker**

### CODE REVIEW

**Comment by backslashbaker** [comment]
@kellyfragkogianni I've added the rest of the pages for you to review

File: roles/sfia/delivery_manager.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+Able to identify, assess & mitigate risks effectively - understands when escalation is required
+
+Finds ways to help our customers beyond just shipping software, ensuring Made Tech teams always demonstrate our mission and values
+
+Upskills and helps customers & team members to resolve complex, often out of the ordinary, issues
+
+### Knowledge
+
+Has a wealth of agile & lean knowledge and has applied both fundamental & advanced principles in different scenarios with little support;
```
Comment (StevenLeighton21) [inline roles/sfia/delivery_manager.md:59]:
```suggestion
Has a wealth of agile & lean knowledge and has applied both fundamental & advanced principles in different scenarios with little support.
```

File: roles/sfia/delivery_manager.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+Able to measure and evaluate delivery outcomes
+
+Remains at the forefront of both agile & delivery topics & new ideas, ensuring they gain new knowledge proactively and regularly
+
+Shares this knowledge with the wider organisation
+
+Shares this knowledge with the customers at every opportunity
+
+### Business Skills
```
Comment (StevenLeighton21) [inline roles/sfia/delivery_manager.md:77]: This section has additional extra lines compared to other sections - suggest keeping them consistent

File: roles/sfia/delivery_manager.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
+# SFIA Role Guidance: Delivery Manager 
+
+[SFIA Level 4: Follow](https://sfia-online.org/en/sfia-7/responsibilities/level-4)
+
+[next &raquo;](senior_delivery_manager.md)
+
+## Required competency for role
+
+### Autonomy
```
Comment (StevenLeighton21) [inline roles/sfia/delivery_manager.md:9]: The various attributes listed have a mix of full stop and no full stop - suggest making them consistent (either way)

**Comment by StevenLeighton21** [comment]
Comments about whitespace and full stop consistency could be extended to all files

**Review by StevenLeighton21 — APPROVED** [review]

File: roles/sfia/delivery_manager.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
+# SFIA Role Guidance: Delivery Manager 
+
+[SFIA Level 4: Follow](https://sfia-online.org/en/sfia-7/responsibilities/level-4)
+
+[next &raquo;](senior_delivery_manager.md)
```
Comment (yusufsheiqh) [inline roles/sfia/delivery_manager.md:5]:
```suggestion
[&laquo; previous](junior_delivery_manager.md) | [next &raquo;](senior_delivery_manager.md)
```

File: roles/sfia/junior_delivery_manager.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
+# SFIA Role Guidance: Junior Delivery Manager 
+
+[SFIA Level 3: Follow](https://sfia-online.org/en/sfia-7/responsibilities/level-3)
+
+[next &raquo;](delivery_manager.md)
```
Comment (yusufsheiqh) [inline roles/sfia/junior_delivery_manager.md:5]:
```suggestion
[&laquo; previous](team_assistant.md) | [next &raquo;](delivery_manager.md)
```

File: roles/sfia/senior_delivery_manager.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
+# SFIA Role Guidance: Senior Delivery Manager 
+
+[SFIA Level 5: Follow](https://sfia-online.org/en/sfia-7/responsibilities/level-5)
+
```
Comment (yusufsheiqh) [inline roles/sfia/senior_delivery_manager.md:4]:
```suggestion

[&laquo; previous](delivery_manager.md)

```

**Review by yusufsheiqh — COMMENTED** [review]
Added a couple of suggestions to go to the previous role pages for the DM career path.

**Pull request merged**